2020-07-08 14:31:39,228 p=andrey u=64384 | PLAY [Test] **************************************************************************************************
2020-07-08 14:31:39,259 p=andrey u=64384 | TASK [routeros : run command on remote devices] **************************************************************
2020-07-08 14:31:39,373 p=andrey u=64384 | fatal: [MikroTik_server]: FAILED! => {"msg": "The conditional check 'ansible_system == 'Linux'' failed. The error was: error while evaluating conditional (ansible_system == 'Linux'): 'ansible_system' is undefined\n\nThe error appears to be in '/Users/andrey/temp/WorkProduction/myansible/NewAnsible/MikroTik/routeros/tasks/main.yml': line 3, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n# tasks file for routeros\n- name: run command on remote devices\n  ^ here\n"}
2020-07-08 14:31:39,375 p=andrey u=64384 | PLAY RECAP ***************************************************************************************************
2020-07-08 14:31:39,376 p=andrey u=64384 | MikroTik_server            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-07-08 14:32:30,233 p=andrey u=64408 | PLAY [Test] **************************************************************************************************
2020-07-08 14:32:30,264 p=andrey u=64408 | TASK [routeros : run command on remote devices] **************************************************************
2020-07-08 14:32:30,324 p=andrey u=64408 | fatal: [MikroTik_server]: FAILED! => {"msg": "Unable to automatically determine host network os. Please manually configure ansible_network_os value for this host"}
2020-07-08 14:32:30,326 p=andrey u=64408 | PLAY RECAP ***************************************************************************************************
2020-07-08 14:32:30,327 p=andrey u=64408 | MikroTik_server            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-07-08 14:32:54,706 p=andrey u=64429 | PLAY [Test] **************************************************************************************************
2020-07-08 14:32:54,730 p=andrey u=64429 | TASK [routeros : run command on remote devices] **************************************************************
2020-07-08 14:32:57,721 p=andrey u=64435 | network_os is set to routeros
2020-07-08 14:33:13,222 p=andrey u=64429 | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.module_utils.connection.ConnectionError: [Errno None] Unable to connect to port 22 on 10.0.7.4
2020-07-08 14:33:13,225 p=andrey u=64429 | fatal: [MikroTik_server]: FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/bin/python"}, "changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/andrey/.ansible/tmp/ansible-local-644294hVvD4/ansible-tmp-1594207977.73-109157617933056/AnsiballZ_routeros_command.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/Users/andrey/.ansible/tmp/ansible-local-644294hVvD4/ansible-tmp-1594207977.73-109157617933056/AnsiballZ_routeros_command.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/andrey/.ansible/tmp/ansible-local-644294hVvD4/ansible-tmp-1594207977.73-109157617933056/AnsiballZ_routeros_command.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.network.routeros.routeros_command', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 188, in run_module\n    fname, loader, pkg_name)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 82, in _run_module_code\n    mod_name, mod_fname, mod_loader, pkg_name)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 72, in _run_code\n    exec code in run_globals\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/modules/network/routeros/routeros_command.py\", line 187, in <module>\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/modules/network/routeros/routeros_command.py\", line 157, in main\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/module_utils/network/routeros/routeros.py\", line 125, in run_commands\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/module_utils/network/routeros/routeros.py\", line 55, in get_connection\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/module_utils/network/routeros/routeros.py\", line 69, in get_capabilities\n  File \"/var/folders/hj/2v16rn4j3qb48_5b7zrnj5mc0000gn/T/ansible_routeros_command_payload_RdLy8f/ansible_routeros_command_payload.zip/ansible/module_utils/connection.py\", line 185, in __rpc__\nansible.module_utils.connection.ConnectionError: [Errno None] Unable to connect to port 22 on 10.0.7.4\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
2020-07-08 14:33:13,230 p=andrey u=64429 | PLAY RECAP ***************************************************************************************************
2020-07-08 14:33:13,230 p=andrey u=64429 | MikroTik_server            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2020-07-08 14:33:13,338 p=andrey u=64443 | shutdown complete
2020-07-08 14:35:34,539 p=andrey u=64514 | PLAY [Test] **************************************************************************************************
2020-07-08 14:35:34,564 p=andrey u=64514 | TASK [routeros : run command on remote devices] **************************************************************
2020-07-08 14:35:37,528 p=andrey u=64521 | network_os is set to routeros
2020-07-08 14:35:50,441 p=andrey u=64514 | ok: [MikroTik_server]
2020-07-08 14:35:50,455 p=andrey u=64514 | TASK [routeros : Display resource_output] ********************************************************************
2020-07-08 14:35:53,075 p=andrey u=64514 | ok: [MikroTik_server] => {
    "resource_output": {
        "ansible_facts": {
            "discovered_interpreter_python": "/usr/bin/python"
        }, 
        "changed": false, 
        "failed": false, 
        "stdout": [
            "uptime: 21w1d4h58m4s\n                  version: 6.46.3 (stable)\n               build-time: Jan/28/2020 10:46:05\n              free-memory: 28.0MiB\n             total-memory: 64.0MiB\n                      cpu: MIPS 74Kc V5.0\n                cpu-count: 1\n            cpu-frequency: 720MHz\n                 cpu-load: 8%\n           free-hdd-space: 3608.0KiB\n          total-hdd-space: 16.0MiB\n  write-sect-since-reboot: 110554\n         write-sect-total: 363908\n               bad-blocks: 0%\n        architecture-name: mipsbe\n               board-name: hEX\n                 platform: MikroTik"
        ], 
        "stdout_lines": [
            [
                "uptime: 21w1d4h58m4s", 
                "                  version: 6.46.3 (stable)", 
                "               build-time: Jan/28/2020 10:46:05", 
                "              free-memory: 28.0MiB", 
                "             total-memory: 64.0MiB", 
                "                      cpu: MIPS 74Kc V5.0", 
                "                cpu-count: 1", 
                "            cpu-frequency: 720MHz", 
                "                 cpu-load: 8%", 
                "           free-hdd-space: 3608.0KiB", 
                "          total-hdd-space: 16.0MiB", 
                "  write-sect-since-reboot: 110554", 
                "         write-sect-total: 363908", 
                "               bad-blocks: 0%", 
                "        architecture-name: mipsbe", 
                "               board-name: hEX", 
                "                 platform: MikroTik"
            ]
        ]
    }
}
2020-07-08 14:35:53,116 p=andrey u=64514 | PLAY RECAP ***************************************************************************************************
2020-07-08 14:35:53,117 p=andrey u=64514 | MikroTik_server            : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
2020-07-08 14:35:53,217 p=andrey u=64528 | shutdown complete
2020-07-08 14:38:47,161 p=andrey u=64615 | PLAY [Test] **************************************************************************************************
2020-07-08 14:38:47,188 p=andrey u=64615 | TASK [routeros : run command on remote devices] **************************************************************
2020-07-08 14:38:50,232 p=andrey u=64622 | network_os is set to routeros
2020-07-08 14:39:02,773 p=andrey u=64615 | ok: [MikroTik_server]
2020-07-08 14:39:02,800 p=andrey u=64615 | PLAY RECAP ***************************************************************************************************
2020-07-08 14:39:02,800 p=andrey u=64615 | MikroTik_server            : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
2020-07-08 14:39:02,899 p=andrey u=64628 | shutdown complete
